print-ldp.c

```diff
@@ -517,6 +529,15 @@ ldp_tlv_print(netdissect_options *ndo,
 	ND_PRINT("\n\t      MTU: %u", GET_BE_U_2(tptr));
 	break;
 
+    case LDP_TLV_DUAL_STACK_CAP:
+        TLV_TCHECK(4);
+	transport_pref = GET_U_1(tptr);
```

Comment:
Indentation is off.

Reply:
Is it ? I am still using emacs default value plus 'c-basic-offset 4' since I have authored the file 21 years ago.
Is there some indent config file or editor settings that you do recommend ?

Reply:
It looks off in GitGub diffs, and loading the file into an editor that can display whitespace (I use Mousepad for that) confirms it:
![Screenshot_2023-08-18_08-26-32](https://github.com/the-tcpdump-group/tcpdump/assets/900115/fa0f2cc9-cc63-47b1-8317-29ea52624735)
The current approach is "try to keep code style consistent per file", and this file seems to assume a tab character for every 8 spaces.